AlphaWallet/Settings/Views/EmptyTableView.swift

```diff
@@ -38,6 +38,11 @@ class EmptyTableView: UIView {
         return nil
     }
 
+    func set(title: String) {
+        self.title = title
+        label.text = title
```

Comment:
Do we need to mark the layout as dirty?

Reply:
I think the label will mark itself dirty.